blockchain/chain/src/store/base_fee.rs

```diff
@@ -10,40 +10,36 @@ use fvm_ipld_encoding::Cbor;
 use fvm_shared::{clock::ChainEpoch, BLOCK_GAS_LIMIT};
 
 /// Used in calculating the base fee change.
-pub const BLOCK_GAS_TARGET: i64 = BLOCK_GAS_LIMIT / 2;
+pub const BLOCK_GAS_TARGET: u64 = BLOCK_GAS_LIMIT as u64 / 2;
```

Comment:
You can use `BLOCK_GAS_LIMIT` from `fvm_shared3`

Reply:
Changed